src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -236,7 +236,7 @@ ucs_status_t uct_rdmacm_cm_ep_resolve_cb(uct_rdmacm_cm_ep_t *cep)
 
     args.field_mask = UCT_CM_EP_RESOLVE_ARGS_FIELD_DEV_NAME;
     uct_rdmacm_cm_id_to_dev_name(cep->id, args.dev_name);
-    return uct_cm_ep_resolve_cb(cep->super.user_data, &args);
+    return uct_cm_ep_resolve_cb(&cep->super, &args);
```

Comment:
this was already merged in another pr right?

Reply:
yes, it was cherry-picked to pass CI, the commit should be dropped on merge automatically or on squash after review if needed